### ISSUE TITLE
PKG_SOURCE_SUBMODULES is nor supported

### ISSUE THREAD

**Issue by winston432** [project-chip/matter-openwrt#17]
Hi, 

I am using openwrt 22.03.4 and encounter "PKG_SOURCE_SUBMODULES" is not supported issue.
Could you advise me what is the openwrt version required to use matter-openwrt ?

best regards,
winston

**Comment by plan44** [comment]
See https://github.com/project-chip/matter-openwrt/issues/2

PKG_SOURCE_SUBMODULES is a patch contributed by project-chip to OpenWrt.
Thing is, it was merged into OpenWrt only after 22.03.4, so when building with 22.03 it is missing.

However, the patch works fine on OpenWrt 22.03, so just apply the [changes from the PR](https://github.com/openwrt/openwrt/pull/13000/commits) on your 22.03 tree.

**Comment by winston432** [comment]
after applying the patch, compilation is successful. I am testing it now.

Thank you.
